### ISSUE TITLE
Examples tools contrast nearly illegible 

### ISSUE THREAD

**Issue by scottmathson** [jamstack/jamstack.org#393]
With a11y and more in mind, we need to adjust the contrast of the _tools_ underneath example listings.

Example - current view:
![/examples/ page contrast issues](https://user-images.githubusercontent.com/25587929/95356200-a82a1400-0883-11eb-8456-ef3bf5011284.png)

**Comment by tabar** [comment]
Yes please, for this one could we also please update the design to match the attached (we may need to tweak it a little to include the additional details)

<img width="1008" alt="Screen Shot 2020-10-07 at 9 11 25 AM" src="https://user-images.githubusercontent.com/13474359/95357791-29ca7380-087d-11eb-81b8-2fcf47000c2f.png">

**Comment by huguestennier** [comment]
Just want to chime in here, I think we should keep the details on the screen for mainly three reasons:

- The name (and domain) of the app/site might be more recognizable than only the screenshot. Easier to scan through all the examples without hovering every projects.
- Showing the tools means I can search quickly through the whole page for keywords like "Gatsby" or "Contentful".
- Better mobile support has there's no hover state
